### PR TITLE
Document local development setup with native tools

### PR DESCRIPTION
**Pull request by olivernybroe**
- [ ] Added or updated tests
- [ ] Documented user facing changes
- [ ] Updated CHANGELOG.md

**Changes**

Add some changes to make local development easier without the usage of a docker container.

**Breaking changes**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -300,3 +300,18 @@ Run the reports that are defined in `phpbench.json` via the command line,
...
 
     vendor/bin/phpbench run --report=ast
+
+## Local setup
+It is also possible to setup the test environment without usage of the docker and make files.
+You would need the following:
+
+- PHP (at least 7.2)
+- Composer
+- A Mysql 8 schema
```
Comment (spawnia) [inline CONTRIBUTING.md:310]: Can you clarify?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -300,3 +300,18 @@ Run the reports that are defined in `phpbench.json` via the command line,
...
+
```
Comment (spawnia) [inline CONTRIBUTING.md:311]: We also have integration tests that require Redis.

**Review by spawnia — COMMENTED** [review]
Looks like this is pretty easy to support, nicely done!

File: CONTRIBUTING.md (hunk 1 above)
Comment (olivernybroe) [reply CONTRIBUTING.md:310]: I could prob. write it a bit better yeah :)

File: CONTRIBUTING.md (hunk 2 above)
Comment (olivernybroe) [reply CONTRIBUTING.md:311]: Ah right. Haven't looked into those yet.

Comment (spawnia) [reply CONTRIBUTING.md:311]: We could also separate them into another testsuite.

Comment (olivernybroe) [reply CONTRIBUTING.md:312]: nah, it was pretty easy to fix it 👍

**Comment by olivernybroe** [comment]
@spawnia alright, redis is also added now 👍 

Think this is rdy.

**Pull request merged**